### DISCUSSION TITLE
How do you use the emulators

### DISCUSSION THREAD

**Discussion by Brady-Shike** [QuiteAFancyEmerald/InvisiProxy#133] [Q&A] [answered]
Plz let me know

**Comment by BinBashBanana** [comment, marked as answer]
How to use the emulators: go find a rom for a game on a rom site, which you can google, and download the rom. Then go to the respective emulator (NES for NES roms, GBA for GBA roms, etc.) Then press the choose file button, or drag the file into the page. It is suggested that you unzip the file if it is a zip file, but you do not have to.

**Reply by ghost** [reply]
Isnt that piracy tho?

**Reply by BinBashBanana** [reply]
Downloading roms *could* be considered piracy but nobody is going to stop you.
